nvflare/private/fed/server/job_runner.py

```diff
@@ -277,8 +285,8 @@ def restore_running_job(self, run_number: str, job_id: str, job_clients, snapsho
             job = job_manager.get_job(jid=job_id, fl_ctx=fl_ctx)
             with self.lock:
                 self.running_jobs[run_number] = job
-        except:
-            self.log_error(fl_ctx, f"Failed to restore the job:{job_id} to the running job table.")
+        except Exception as e:
+            self.log_error(fl_ctx, f"Failed to restore the job: {job_id} to the running job table: {e}.")
```

Comment:
do you need the stack trace ?  exec_info = True ?

Reply:
this is not logger, we could add that into general log_error though